client/client.go

```diff
@@ -30,6 +30,11 @@ func attentiveBackoff(min, max time.Duration, attemptNum int, resp *http.Respons
 
 			timeToWait := time.Until(retryAfterDate)
 
+			if timeToWait < 1 {
```

Comment:
`timeToWait` is a `time.Duration`, which is an int in nanoseconds. I think this line should be `if timeToWait < 1 * time.Second {`

Reply:
Oop! true.

Reply:
I can also just return 1*time.Second as well I think